content/python/concepts/strings/terms/casefold/casefold.md

```diff
@@ -39,3 +39,13 @@ text = "this Should All Be In Lower Case!"
 x = text.casefold()
 print(x)
 ```
+
+## Codebyte Example
```

Comment:
```suggestion
## Example 3
```

All the other examples use codebyte snippets.

Reply:
Whoops, that was my bad. For some reason thought the other examples were just snippets 😅 thanks for catching that